src/core/hle/libraries/libkernel/thread_management.cpp

```diff
@@ -8,22 +8,28 @@
 
 namespace Core::Libraries::LibKernel {
 
-thread_local PthreadInternal g_pthread_self{};
+thread_local ScePthread g_pthread_self{};
```

Comment:
Does this need to be a pointer? The old way avoided a call to new which you aren't freeing.